iot-e2e-tests/common/pom.xml

```diff
@@ -139,6 +139,7 @@
                     </execution>
                 </executions>
                 <configuration>
+                    <rerunFailingTestsCount>2</rerunFailingTestsCount>
```

Comment:
Is this more of a temporary fix while service issue is resolved?

Reply:
Hopefully, yes.